crates/rpc-types/src/eth/transaction/request.rs

```diff
@@ -62,6 +64,17 @@ impl From<OptimismTransactionFields> for OtherFields {
     }
 }
 
+/// This represents a set of blobs, and its corresponding commitments and proofs.
+#[derive(Clone, Debug, PartialEq, Eq, Default, Serialize, Deserialize)]
+#[repr(C)]
+pub struct BlobTransactionSidecar {
+    /// The blob data.
+    pub blobs: Vec<Blob>,
+    /// The blob commitments.
+    pub commitments: Vec<Bytes48>,
+    /// The blob proofs.
+    pub proofs: Vec<Bytes48>,
+}
```

Comment:
I have added BlobTransactionSidecar here which is the type for sidecar field for EIP-4844 transactions, in TransactionRequest, is this the correct approach or should I do this using other::OtherFields?